### ISSUE TITLE
Register option still shows

### ISSUE THREAD

**Issue by Baltimorepc** [immense/Remotely#765]
According to the instuctions once you setup the register option is supposed to go away so no one can register on there own. i wouldnt mind it emailing me asking for permission. But anyone can register right now and use it. I dont want that how can i fix it

**Comment by CFox243** [comment]
do you have max organization count  >0?

**Comment by Baltimorepc** [comment]
I don't recall that option if there was one I will double check. 
I checked the variables and don’t see that. Could you give me the info so I can add it please 


Sent from Yahoo Mail for iPhone


On Monday, November 6, 2023, 8:24 AM, CFox243 ***@***.***> wrote:




do you have max organization count >0?

**Comment by CFox243** [comment]
![chrome_FJsXSpLX5i](https://github.com/immense/Remotely/assets/128409855/db071567-f4b2-45f1-9440-7e79ddaf5649)

**Comment by Baltimorepc** [comment]
It was set to -1 guess by default as I never changed that

**Comment by Baltimorepc** [comment]
I just changed it to 0 thanks will test 


Sent from Yahoo Mail for iPhone


On Monday, November 6, 2023, 8:24 AM, CFox243 ***@***.***> wrote:




do you have max organization count >0?

**Comment by CFox243** [comment]
MaxOrganizationCount: By default, one organization can exist on the server, which is created automatically when the first account is registered. Afterward, self-registration will be disabled.

    Set this to -1 or increase it to a specific number to allow multi-tenancy.

**Comment by Baltimorepc** [comment]
Thank you it worked I looked everywhere trying to figure this out and just couldn’t locate this information you provided. Thank you very much 


Sent from Yahoo Mail for iPhone


On Monday, November 6, 2023, 11:46 AM, CFox243 ***@***.***> wrote:




MaxOrganizationCount: By default, one organization can exist on the server, which is created automatically when the first account is registered. Afterward, self-registration will be disabled.
Set this to -1 or increase it to a specific number to allow multi-tenancy.

**Comment by Baltimorepc** [comment]
If I may, is there a way to remote control machines that have resident agents or just a certain group that I can remote without requesting access. As I remote into my own machines and only way I can is if I have someone allow the connection at home. I don’t want to turn off the remote control notify user as I want to keep that on 


Sent from Yahoo Mail for iPhone


On Monday, November 6, 2023, 11:46 AM, CFox243 ***@***.***> wrote:




MaxOrganizationCount: By default, one organization can exist on the server, which is created automatically when the first account is registered. Afterward, self-registration will be disabled.
Set this to -1 or increase it to a specific number to allow multi-tenancy.

**Comment by CFox243** [comment]
I do not believe there is an option to enable and disable enforced attended access for certain systems.
the only option would be to change the max organization count to 2 and create a second instance for unattended. and add the personal system to that one.

I could be mistaken, but I do not believe that's the case.

**Comment by Baltimorepc** [comment]
Thank you now my issue is even after a user allows me access they still need to be there to click allow or yes when i am uninstalling things its like i am not able to see admin options. As this is a real down fall if a user still needs to sit around its bad enough we cant do reboots with the portable verison and now cant even do admin rights i hope they can fix this
